runregistry-rest/entrypoint.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-
 cd $(dirname $0)
 source ../entrypoint_functions.sh
 
-ensure_required_variables "DB_HOSTNAME DB_PORT DB_NAME DB_USERNAME DB_PASSWORD"
-
-
-mkdir --mode=1777 uploads
+ensure_required_variables "DATABASE_URI"
 
-export LD_LIBRARY_PATH=/usr/lib/oracle/12.1/client64/lib/:$LD_LIBRARY_PATH
+if [[ ! -d /uploads ]]; then
```

Comment:
I think this path needs to be fiddled with

Reply:
What is the new path then?

Reply:
https://github.com/DUNE-DAQ/microservices/commit/15a1b5b5955c397e98c86577d0ffdabc9c316334  should be sufficient.  The program has already run `cd` at the dir test point.